### DISCUSSION TITLE
"View in your space" from google seems like is not using iOS Quick Look app

### DISCUSSION THREAD

**Discussion by hbaugmentedreality** [google/model-viewer#2662] [Q&A]
Hi guys.   I'm not very happy with the AR viewer in the iOS device, using Quick Look app (and .USDZ file) mainly because the lights estimation doesn't work fine and the models shows up very dark. 
I have seen that the google "View in your space" (to see animals or other 3D models in AR) doesn't use apparently Quick Look app. 
Is it possible that those aplications uses another ar viewer app?   Is there any way to improve the AR model quality (mainly the lights?).
I have attached an image where you can see it doens't look like Quick Look used by model-viewer (in a iOS device - iPhone)

Any comments or advice are welcome!!
Thanks!!!

![WhatsApp Image 2021-08-06 at 11 36 55 (1)](https://user-images.githubusercontent.com/73670108/128527146-4ae0a4cf-09df-47d8-914b-cd32ddf57239.jpeg)

**Comment by elalish** [comment]
Indeed, you are correct. This "other experience on iOS" is in fact Scene Viewer in iGA (the Google App for iOS). On Android Scene Viewer is part of AGA (Android Google App). Unfortunately, from a general webpage MV does not have the ability to launch iGA with an arbitrary 3D model currently, which is why we still use Quick Look. Also, many iOS users don't have iGA installed anyway. 

It's also possible there has been some problem in converting to USDZ. Have you tried our auto-conversion feature? Just specify `ar-modes="webxr scene-viewer quick-look" and no `ios-src`. You'll also want to make sure that your GLB is not using uncommon things like SpecGloss, vertex colors, multiple UVs, etc, or it may not be convertible to USDZ.

**Reply by hbaugmentedreality** [reply]
Thanks @elalish for your answer.   I have tried the auto-conversion but it didn't work.   Does it work with glb with animations?

**Reply by elalish** [reply]
No, not yet.
